packages/web/src/pages/deactivate-account-page/DeactivateAccountPage.tsx

```diff
@@ -86,22 +86,18 @@ export const DeactivateAcccountPageContents = ({
       )}
       <div className={styles.buttons}>
         <Button
-          className={cn(styles.button, {
-            [styles.buttonDanger]: !(isLoading && isMobile)
-          })}
-          text={messages.buttonDeactivate}
-          type={
-            isLoading && isMobile ? ButtonType.DISABLED : ButtonType.PRIMARY_ALT
-          }
+          variant='destructive'
+          isLoading={isLoading}
           onClick={openConfirmation}
-        />
+          fullWidth={isMobile}
+        >
+          {messages.buttonDeactivate}
+        </Button>
         <Button
-          className={styles.button}
-          text={messages.buttonSafety}
-          type={
-            isLoading && isMobile ? ButtonType.DISABLED : ButtonType.PRIMARY_ALT
-          }
+          variant='secondary'
+          isLoading={isLoading}
           onClick={goToSafety}
+          fullWidth={isMobile}
         />
```

Comment:
You forgot the text here (messages.buttonSafety as the child)

Reply:
nice find!!!